src/commands/test/parse-options.js

```diff
@@ -27,7 +27,7 @@ function parseOptions(args, config) {
     chromeLoadTimeout: parseInt($('chromeLoadTimeout'), 10),
     chromeRetries: parseInt($('chromeRetries'), 10),
     chromeSelector: $('chromeSelector'),
-    chromeTolerance: parseFloat($('chromeTolerance'), 10),
+    tolerance: parseFloat($('tolerance')),
```

Comment:
Removed the `10`, assuming it was there accidentally. `parseInt` takes a `radix` parameter. `parseFloat` does not 😸